src/Our.Umbraco.InnerContent/Web/UI/App_Plugins/InnerContent/js/innercontent.controllers.js

```diff
@@ -118,9 +122,9 @@ angular.module('umbraco.directives').directive('innerContentOverlay', [
             scope.config.editorModels = scope.config.editorModels || {};
             scope.currentItem = null;
 
-            var getContentType = function (alias) {
+            var getContentType = function (guid) {
                 return _.find(scope.config.contentTypes, function (ct) {
-                    return ct.icContentTypeAlias === alias;
+                    return ct.icContentTypeGuid == guid;
```

Comment:
Following on from a previous comment, should this be a triple-equals? or are we sure that the values are string types here? (I mean, I assume they are, but just asking the question)